src/main/java/org/broadinstitute/hellbender/tools/variantdb/IngestUtils.java

```diff
@@ -73,15 +73,14 @@ public static String getSampleId(final String sampleName, final File sampleMap)
     // Since tables have a limited number of samples (default is 4k)
     public static int getTableNumber(String sampleId, int sampleMod) { // this is based on sample id
         // sample ids 1-4000 will go in directory 001
-        int sampleIdInt = Integer.valueOf(sampleId); // TODO--should sampleId just get refactored as a long?
+        long sampleIdInt = Long.valueOf(sampleId); // TODO--should sampleId just get refactored as a long?
```

Comment:
rename variable, maybe just sampleId and change the parameter to the function to be sampleIdString.  MAybe remove the TODO